atmos.yaml

```diff
@@ -281,10 +283,11 @@ schemas:
   # https://json-schema.org/draft/2020-12/release-notes
   # https://www.schemastore.org/json
   # https://github.com/SchemaStore/schemastore
-  atmos:
+  # atmos:
```

Comment:
Why is this commented out?

Reply:
Something is wrong with the white space of the comments then, because it appears atmos is at the same level as the subkeys.